v2/concordium/types.proto

```diff
@@ -1947,8 +1960,11 @@ message BlockItemSummary {
     AccountTransactionDetails account_transaction = 4;
     // Details about an account creation.
     AccountCreationDetails account_creation = 5;
-    // Details about a chain update.
+    // Details about a successful chain update.
     UpdateDetails update = 6;
+    // Details about a rejected chain update.
+    // This only applies to protocol-level token chain governance updates.
+    FailedUpdateDetails failed_update = 7;
```

Comment:
Since there is no one paying for this transaction, do we really need it to be represented in the block summary?
Wouldn't the node just throw it away?